dev/lib/product_taxonomy/cli.rb

```diff
@@ -4,7 +4,10 @@ module ProductTaxonomy
   class Cli < Thor
     desc "dist", "Generate the taxonomy distribution"
     def dist
-      puts "TODO"
+      seconds = Benchmark.realtime do
```

Comment:
Good idea, it could also be helpful to have a benchmark suite that runs along with unit tests to detect performance regressions. I'll aim to tackle this in an upcoming PR.